app/src/pages/inside/testItemPage/notFound.jsx

```diff
@@ -0,0 +1,27 @@
+import Parser from 'html-react-parser';
+import { injectIntl, defineMessages } from 'react-intl';
+import classNames from 'classnames/bind';
+import ErrorIcon from 'common/img/error-inline.svg';
+import styles from './notFound.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  checkQuery: {
+    id: 'LaunchesPage.checkQuery',
+    defaultMessage: 'Failed to load data for launches table',
+  },
+  notFound: {
+    id: 'LaunchesPage.notFound',
+    defaultMessage: 'Launch is not found',
+  },
+});
+
+export const NotFound = injectIntl(({ intl: { formatMessage } }) => (
+  <div className={cx('launch-not-found')}>
+    <p className={cx('launch-not-found-text')}>
```

Comment:
You can use here `NoItemMessage` component from `components/main/noItemMessage`

Reply:
It has only one message parameter, but the requirement is 2 texts.

![image-epmrpp-35499](https://user-images.githubusercontent.com/18571747/52559644-2bb50880-2df6-11e9-9878-8c49ffe0c93b.png)

Reply:
Fixed